src/Symfony/Component/Security/Core/Authorization/AccessDecision.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Authorization;
+
+use Symfony\Component\Security\Core\Authorization\Voter\AccessTrait;
+use Symfony\Component\Security\Core\Authorization\Voter\Vote;
+use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+
+class AccessDecision
+{
+    use AccessTrait;
+
+    /** @var Vote[] */
+    private $votes;
+
+    private function __construct(int $access, array $votes = [])
```

Comment:
I would call it $accessType

$access for me relates to bool logic etc

Wdyt?

Reply:
The related constante is called ACCESS_GRANTED - not ACCESS_TYPE_GRANTED. It looks fine to me imo.